bridge/client/app/_components/ktb-subscription-item/ktb-subscription-item.component.html

```diff
@@ -1,35 +1,41 @@
-<div fxLayout="column" *ngIf="subscription">
-  <div fxLayout="row">
-    <input fxFlex="200px" class="mr-2 item" placeholder="Subscription name" dtInput [(ngModel)]="subscription.name" required>
-    <span fxFlex="100px" class="bold mr-1">subscribes to:</span>
-    <dt-select [(ngModel)]="subscription.event" class="mr-2 item" placeholder="Choose your task" aria-label="Choose your task" required>
-      <dt-option *ngFor="let task of tasks" [value]="task" [textContent]="task"></dt-option>
-    </dt-select>
-    <dt-filter-field
-      [dataSource]="_dataSource"
-      [filters]="subscription.getFilter(_dataSource.data)"
-      (filterChanges)="filterChanged(subscription, $event)"
-      label="Filter by stage and service"
-      aria-label="Filter by stage and service"
-      clearAllLabel="Clear all"
-    ></dt-filter-field>
+<ng-container *ngIf="project && subscription as subscription">
+  <span class="pl-3 pr-4" [textContent]="name"></span>
+  <span class="bold pl-3 pr-2">subscribes to:</span>
+  <span class="mr-1 pr-3" [textContent]="subscription.event"></span>
+  <span class="bold pr-2">in projects:</span>
+  <span class="mr-1 pr-3" [textContent]="subscription.formatFilter('projects') || 'all'"></span>
+  <span class="bold pr-2">in stages:</span>
+  <span class="mr-1 pr-3" [textContent]="subscription.formatFilter('stages') || 'all'"></span>
+  <span class="bold pr-2">for services:</span>
+  <span class="mr-1 pr-3" [textContent]="subscription.formatFilter('services') || 'all'"></span>
+  <div>
+    <ng-container *ngIf="subscription.id, else oldSubscription">
+      <button dt-icon-button variant="nested" uitestid="subscriptionEditButton" title="Edit" (click)="editSubscription(subscription)">
+        <dt-icon name="edit1"></dt-icon>
+      </button>
+      <button dt-icon-button variant="nested" uitestid="subscriptionDeleteButton" title="Delete" (click)="triggerDeleteSubscription(subscription)">
+        <dt-icon name="delete"></dt-icon>
+      </button>
+    </ng-container>
+    <ng-template #oldSubscription>
+      <button dt-icon-button variant="nested" uitestid="subscriptionEditButton" class="disabled" [dtOverlay]="updateDistributor">
+        <dt-icon name="edit1"></dt-icon>
+      </button>
+      <button dt-icon-button variant="nested" uitestid="subscriptionDeleteButton" class="disabled" [dtOverlay]="updateDistributor">
+        <dt-icon name="delete"></dt-icon>
+      </button>
+    </ng-template>
   </div>
 
+  <ktb-delete-confirmation
+    type="subscription"
+    [dialogState]="deleteState"
+    [deleteMessage]="subscription.isGlobal ? 'Deleting this subscription will affect all projects!' : undefined"
+    [name]="name"
+    (confirmClicked)="deleteSubscription()"
+  ></ktb-delete-confirmation>
+  <ng-template #updateDistributor>
+    Action denied. Please update your distributor to version 0.9.0 or higher
```

Comment:
Rephrase: "Please update your distributor to version 0.9.0 or higher for editing your subscriptions."